packages/modules/bookmark/src/module.ts

```diff
@@ -0,0 +1,41 @@
+import { Module, ModulesInstance } from '@equinor/fusion-framework-module';
+import { EventModule } from '@equinor/fusion-framework-module-event';
+import { ServicesModule } from '@equinor/fusion-framework-module-services';
+import { AppModule } from '@equinor/fusion-framework-module-app';
+import { ContextModule } from '@equinor/fusion-framework-module-context';
+import { NavigationModule } from '@equinor/fusion-framework-module-navigation';
```

Comment:
is this needed?

Reply:
@odinr  I am using some of them in configuration. so i would say they are needed. I can delete Navigation Module